src/Files.App/UserControls/Widgets/FileTagsWidget.xaml.cs

```diff
@@ -0,0 +1,59 @@
+﻿using CommunityToolkit.Mvvm.DependencyInjection;
+using Files.App.Extensions;
+using Files.App.ViewModels.Widgets;
+using Files.Backend.Services.Settings;
+using Files.Backend.ViewModels.Widgets.FileTagsWidget;
+using Microsoft.UI.Xaml.Controls;
+using System;
+using System.Threading.Tasks;
+
+// The User Control item template is documented at https://go.microsoft.com/fwlink/?LinkId=234236
+
+namespace Files.App.UserControls.Widgets
+{
+	public sealed partial class FileTagsWidget : UserControl, IWidgetItemModel
+	{
+		private IUserSettingsService UserSettingsService { get; } = Ioc.Default.GetRequiredService<IUserSettingsService>();
+
```

Comment:
Can you make this field readonly and rename it `userSettingsService`?

Reply:
Is there any reason we want to adhere services to be fields? For me, it makes more sense that a service is a property since it can be easily differenciated from rest of the fields which should be injected through a constructor

Reply:
I would make services as fields to ensure that no one edits them after they have been initialized.<br/> I also believe that the suffix `Service` is sufficient to differentiate them from other fields

Reply:
`{ get; }` only properties are readonly by default